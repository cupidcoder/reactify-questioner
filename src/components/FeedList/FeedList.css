.FeedList h5 {
  font-size: 1.4rem;
  padding: 1rem;
  margin-top: 1rem;
  margin-left: 2rem;
}
