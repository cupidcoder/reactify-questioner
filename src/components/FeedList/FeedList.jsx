import React from 'react';
import QuestionList from '../QuestionList/QuestionList';
import style from './FeedList.css';

const FeedList = () => {
  const meetups = [
    { no: 1, meetup: 'NodeJs' },
    { no: 2, meetup: 'Artificial Intelligence' }
  ];
  return (
    <div className={style.FeedList}>
      {meetups.map(meetup => (
        <div key={meetup.no}>
          <h5>{meetup.meetup}</h5>
          <QuestionList />
        </div>
      ))}
    </div>
  );
};

export default FeedList;
