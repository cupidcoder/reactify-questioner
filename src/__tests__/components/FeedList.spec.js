import React from 'react';
import { mount } from 'enzyme';
import FeedList from '../../components/FeedList/FeedList';

describe('Feed List', () => {
  it('Should correctly save snapshot of the FeedList component', () => {
    const FeedListWrapper = mount(<FeedList />);
    expect(FeedListWrapper).toMatchSnapshot();
  });
});
