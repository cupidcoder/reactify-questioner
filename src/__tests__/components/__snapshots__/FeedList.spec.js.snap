// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed List Should correctly save snapshot of the FeedList component 1`] = `ReactWrapper {}`;
